tests/test_funcs.py

```diff
@@ -197,7 +198,7 @@ def test_kron_smoke(self):
             ((2, 0, 0, 2), (2, 0, 2)),
         ],
     )
-    def test_kron_shape(self, shape_a, shape_b):
+    def test_kron_shape(self, shape_a: tuple[int], shape_b: tuple[int]):
```

Comment:
a `tuple[int]` only accepts tuples of length 1

```suggestion
    def test_kron_shape(self, shape_a: tuple[int, ...], shape_b: tuple[int, ...]):
```

Reply:
i HATE tuple semantics